<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignatureDesigner</class>
 <widget class="QMainWindow" name="SignatureDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignatureDesigner</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionRedo"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSet_Order">
    <property name="title">
     <string>Set Order</string>
    </property>
    <addaction name="actionOrder_1"/>
    <addaction name="actionOrder_2"/>
    <addaction name="actionOrder_3"/>
   </widget>
   <widget class="QMenu" name="menuSet_Width">
    <property name="title">
     <string>Set Width</string>
    </property>
    <addaction name="action1_pixel"/>
    <addaction name="action3_pixel"/>
    <addaction name="action5_pixel"/>
    <addaction name="action7_pixel"/>
   </widget>
   <widget class="QMenu" name="menuSet_Color">
    <property name="title">
     <string>Set Color</string>
    </property>
    <addaction name="actionBlack"/>
    <addaction name="actionRed"/>
    <addaction name="actionBlue"/>
    <addaction name="actionYellow"/>
   </widget>
   <widget class="QMenu" name="menuPicture">
    <property name="title">
     <string>Picture</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionSet_Path"/>
    <addaction name="actionClear_Picture"/>
    <addaction name="actionClear_Curve"/>
    <addaction name="actionClear_All"/>
   </widget>
   <widget class="QMenu" name="menuOther_Options">
    <property name="title">
     <string>Curve Options</string>
    </property>
    <addaction name="actionOpen_Curve"/>
    <addaction name="actionClose_Curve"/>
   </widget>
   <widget class="QMenu" name="menuPath_Filling">
    <property name="title">
     <string>Path Filling</string>
    </property>
    <addaction name="actionParallel_Contour"/>
    <addaction name="actionZigzag"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Offset"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSet_Order"/>
   <addaction name="menuSet_Width"/>
   <addaction name="menuSet_Color"/>
   <addaction name="menuPicture"/>
   <addaction name="menuOther_Options"/>
   <addaction name="menuPath_Filling"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionOrder_1">
   <property name="text">
    <string>Order 1</string>
   </property>
  </action>
  <action name="actionOrder_2">
   <property name="text">
    <string>Order 2</string>
   </property>
  </action>
  <action name="actionOrder_3">
   <property name="text">
    <string>Order 3</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionYellow">
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="action1_pixel">
   <property name="text">
    <string>1 pixel</string>
   </property>
  </action>
  <action name="action3_pixel">
   <property name="text">
    <string>3 pixel</string>
   </property>
  </action>
  <action name="action5_pixel">
   <property name="text">
    <string>5 pixel</string>
   </property>
  </action>
  <action name="action7_pixel">
   <property name="text">
    <string>7 pixel</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionClear_Picture">
   <property name="text">
    <string>Clear Picture</string>
   </property>
  </action>
  <action name="actionClose_Curve">
   <property name="text">
    <string>Close Curve</string>
   </property>
  </action>
  <action name="actionOpen_Curve">
   <property name="text">
    <string>Open Curve</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionClear_Curve">
   <property name="text">
    <string>Clear Curve</string>
   </property>
  </action>
  <action name="actionParallel_Contour">
   <property name="text">
    <string>Parallel Contour</string>
   </property>
  </action>
  <action name="actionSet_Offset">
   <property name="text">
    <string>Set Offset</string>
   </property>
  </action>
  <action name="actionZigzag">
   <property name="text">
    <string>Zigzag</string>
   </property>
  </action>
  <action name="actionSet_Path">
   <property name="text">
    <string>Set Path</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SignatureDesigner.qrc"/>
 </resources>
 <connections/>
</ui>
